packages/hub/src/components/EntityLayout.tsx

```diff
@@ -1,21 +1,19 @@
 import { FC, ReactNode } from "react";
 import { clsx } from "clsx";
 
-import { EntityInfo } from "./EntityInfo";
+import { EntityInfo, entityNode } from "./EntityInfo";
+
+export type { entityNode };
 
 type Props = {
-  slug: string;
-  username: string;
-  homepageUrl: string;
+  nodes?: entityNode[];
```

Comment:
Why is this optional? You always pass `nodes`, AFAICS.

Reply:
Good point, will change